pulseguardian/web.py

```diff
@@ -350,38 +351,79 @@ def update_info():
     pulse_username = request.form['pulse-user']
     new_password = request.form['new-password']
     password_verification = request.form['new-password-verification']
+    new_owners = _clean_owners_str(request.form['owners-list'])
 
     try:
         pulse_user = PulseUser.query.filter(
             PulseUser.username == pulse_username).one()
     except sqlalchemy.orm.exc.NoResultFound:
-        return profile(messages=["Invalid user."])
-
-    if pulse_user.owner != g.user:
-        return profile(messages=["Invalid user."])
-
-    if not new_password:
-        return profile(messages=["You didn't enter a new password."])
-
-    if new_password != password_verification:
-        return profile(error="Password verification doesn't match the "
-                       "password.")
-
-    if not PulseUser.strong_password(new_password):
-        return profile(error="Your password must contain a mix of "
-                       "letters and numerical characters and be at "
-                       "least 6 characters long.")
+        return profile(
+            messages=["Pulse user {} not found.".format(pulse_username)])
```

Comment:
Can you double-check that this is sanitized, i.e., that you can't send HTML that will be rendered when returned? I don't think it's a big deal, since it's a POSTed form, but best to be on the safe side.

Please check each place where you're returning variable data, in this function and elsewhere.